### ISSUE TITLE
Doesn't compile as a dependency of a dependency

### ISSUE THREAD

**Issue by iopq** [rust-lang/num#88]
I can't compile my crate because a dependency uses `num` which doesn't compile

```
53:37 error: use of unstable library feature 'float_from_str_radix': recently moved API
             <$t>::from_str_radix(s, radix)
                   ^~~~~~~~~~~~~~
57:2 note: in expansion of float_trait_impl!
60:36 note: expansion site
 53:37 help: add #![feature(float_from_str_radix)] to the crate attributes to enable
 53:37 error: use of unstable library feature 'float_from_str_radix': recently moved API
             <$t>::from_str_radix(s, radix)
                   ^~~~~~~~~~~~~~
57:2 note: in expansion of float_trait_impl!
60:36 note: expansion site
 53:37 help: add #![feature(float_from_str_radix)] to the crate attributes to enable
```

**Comment by mitchmindtree** [comment]
The issue is that the `.from_str_radix` method has been "moved" - the std docs mention this but they do not mention where it has been moved too.

**Comment by iopq** [comment]
this is the commit: https://github.com/rust-lang/rust/commit/a568a7f9f2eb3fa3f3e049df288ef0ad32cc7881

**Comment by mitchmindtree** [comment]
Think I've fixed it at #87
